components/_share.scss

```diff
@@ -0,0 +1,170 @@
+.sharing-info {
+  clear: both;
+  margin-top: 50px;
+
+  @include span-columns(8 of 12);
+  @include shift(2);
+
+  h2 {
+    margin-bottom: $base-font-size;
+  }
+}
+
+.sharing-box {
+  clear: both;
+  border: 2px solid #F0F0F1;
+  margin: auto;
+  padding: 20px;
+  background-color: #FFFFFF;
```

Comment:
Color literals like `#FFFFFF` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FFFFFF` should be written as `#ffffff`